XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs

```diff
@@ -385,6 +386,11 @@ private void UpdateSortAndFilter()
                 filteredTypes.OrderBy(i => i.IngList.Count());
             // Re-initialize MyItemsSource object with this newly filtered data
             filteredRecipeData.InitializeCollection(sortedFilteredTypes);
+
+            var peer = FrameworkElementAutomationPeer.FromElement(VariedImageSizeRepeater);
+
+            peer.RaiseNotificationEvent(AutomationNotificationKind.Other, AutomationNotificationProcessing.ImportantMostRecent, $"Filtered recipes, {sortedFilteredTypes.Count()} results.", "RecipesFilteredNotificationActivityId");
```

Comment:
Does this notification/event only get called once?

Reply:
It get's called every time we update the item collection of the ItemsRepeater so every time you update the search criteria.